code/src/reducers/games.js

```diff
@@ -0,0 +1,95 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const games  = createSlice({
```

Comment:
You could do an export default of games instead to make it more clear in the components which one is the main slice.